core/services/ocr2/plugins/mercury/fixtures/domain.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCswr7N//BPhAkF
+mtvwOge2j536x8IJygZZNccLmOGXG2aj0FI9xD2kBBlzpk+/Ma5WIGbcVgcum49m
+b5+upEY8nmZK74yGluNKUi2Zoiw3f7uBA2EwFI4irfDT3WUGr7N54hdIWi0GgegP
+KjAqkPfNrd3ZP3mJcTfDR0KbgT/Gv65MLHEPkdkh7dHlsdSIRVKkfrAZ+Vr8uSZq
+TZlapwZHn1RaeYdta11zvFwEyRxmHFDNZUpwpr8pk8ytjQwgkIe9lzIWN8esO5An
+7yT9AICEXcumVjvU6DTXOwtexTPfegKN7S4eFU4b5ctY/TLaxws0xbxL+/tAytNf
+qcFou22fAgMBAAECggEAD9723GyOInVExukKb4VFPsuZfc0VfFoSPzvQ2gs6HkGm
+1OxxwbmM2CEwSA7tSz6V1NUL3g4L1tz0+Dok90xMJOGAZZNK25/HqFo9qjGF0mC/
+qa3VIy+gPHtc9nm5ADuVR2CH4bO3EzP396dZNBqL9ASXBDrv6rp7SaClnoI/M6jd
+lreoedH5TxDQ+tS+smmOhOVXYr8H/y3I2Kt4GrcuvZVp7ftL0zcN7q2l4TAvPE/B
+a/t8Vp4kdMAOHMCAGXs4YeQ+kDpgKzUrDrRhpxoC4EBkHZNDcd4ww9Bb9hSFNzFk
+oNZYaB6Lxm7Gw7cVtz6IYCJhrIQEj7J5ddCR3/CesQKBgQDicvOhUaERiyjKgWib
+mzinGkAgPFTupKH+VEFcQww7VuUS7lyRv2LFaUB2iJTKt39zipht1p3ZZDIGdoEL
+J3lMqw3wPMhkhrnu509ma+0XWJG/4mW/tVPfeGLhWQb8FwR41BT2mVCw6jEUpApT
+rjFq5AFPmxR5fx39YRDWkToRmwKBgQDDTjZ+NdtUbalVBTG271z3Hyf1JSwAs806
+H6wcLHhKGZGZPPLYNKv+KFKZiXCJuyDUfAzwsfelNzq0PLfymhnmsYG52YIxVJ5c
+vl2uX9LH853TNwaGDabCpbEj4X7OxTOjp0rwsESpOnEOP8w7HgDvX+3Rrk89HNni
+78yM0neGTQKBgQCYiz87iLWsD8ZmxE4npKTBTJmuHXZJXHYT0cZF1JTE1IB0BEYv
+mHF88q4OK2uwM3ST7OVJ+K1U6H4ahHHhhbHcx84X65pCvc869G80W7fXZj6ouGY6
+gngBriNOEp716ruEebm1dQo6Y0i00txyCoIXs4h+i8V7Ivqc2WpyYFpxyQKBgQCx
+lsTmLoKpWq3GqVpzRWP7MVFcD9jbKqNKXtJZK/aVKnrCJDGNdxeNm4zAH8E8+/L2
+Q3ZgxvKwQHAYw71f09AJfQ7At48y3AbDvjXhATDgsByqmjZMXs1r4M4aGkj8K0Sx
+YsU55o4IcyOqGUjT2IxxHYFaUG2s3Dcjq3kL87TLYQKBgGcBQz2mhwaTAMSpo6n5
+QWtYYmLmE3DCsAH1l0GcgSNYY4ZsZ1066xw7t1BABWScInlkA3BbeQJEYclhVli/
+9cRv+xUORGHQ08btRxLdDKX0qw50LmRJONYjHDPQnUJSxY2pWGi9JBIaE5uMAqBl
+FxIhwpSwH/dcU8gRXU4yllLA
+-----END PRIVATE KEY-----
```

Comment:
Did you mean to commit a private key?

Reply:
Yep, these are just test fixtures.

Reply:
Can the filename be changed to something like `domain_test.key` to emphasize this is for tests?